app/models/branding_group.rb

```diff
@@ -12,6 +12,9 @@ class BrandingGroup
   field :font_color, type: String, default: '#333333'
   field :feature_flags, type: Hash, default: {}
 
+  # list of facets to show for this branding group (will restrict to only provided identifiers, if present)
+  field :facet_list, type: Array, default: []
```

Comment:
this is good.  I can't think of a case where we'd want to have a branding group with no facets (but still have advanced search), so I like how you're using the empty array as the default so we don't have to have nil checks everywhere